chia/full_node/mempool_manager.py

```diff
@@ -435,12 +442,18 @@ async def validate_spend_bundle(
                 coin_id,
                 EligibilityAndAdditions(is_eligible_for_dedup=False, spend_additions=[], is_eligible_for_ff=False),
             )
-            mark_as_fast_forward = eligibility_info.is_eligible_for_ff and supports_fast_forward(coin_spend)
+            mark_as_fast_forward = (
+                eligibility_info.is_eligible_for_ff
+                and eligibility_info.ff_puzzle_hash is not None
```

Comment:
this suggests that `ff_puzzle_hash` may be disconnected from `is_eligible_for_ff`. I wouldn't have thought so